Jenkinsfile

```diff
@@ -2,12 +2,17 @@
 node {
   stage('checkout sources') {
         // You should change this to be the appropriate thing
-        git url: 'https://github.com/jschmersal-cscc/special-topics-labs-quality'
+        git url: 'https://github.com/Felixda1/special-topics-labs-ci.git'
   }
 
   stage('Build') {
-    // you should build this repo with a maven build step here
-    echo "hello"
+    withMaven (maven: 'maven3') {
+              sh "mvn package"
+            }
+  }
+      post {
```

Comment:
It turns out there are two types of DSL jenkins supports.  `post`  happens to not be supported in the one you're using.  If you just remove the `post` and `always` blocks and have the straight `junit` line your lab would work perfectly.

You can fix that up and get a 20/20 on this lab, or just submit as is and between your two labs you'll get a passing grade.  Either way I just need to see it in blackboard and can then post your final grade!